pkg/apis/core/register.go

```diff
@@ -60,7 +60,7 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&ServiceProxyOptions{},
 		&NodeList{},
 		&Node{},
-		&NodeConfigSource{},
+		&SerializedNodeConfigSource{},
```

Comment:
this should be able to be internal to the kubelet, right? not defined/registered for the entire `v1` API group

Reply:
Hmm. It felt appropriate here, since the type it wraps is also here...
Do we expect the Kubelet to be the only component that ever wants to serialize this?